koschei/models.py

```diff
@@ -442,6 +442,11 @@ class Build(Base):
     version = Column(String)
     release = Column(String)
     repo_id = Column(Integer)
+    # whether this build is the last complete build for corresponding package
+    # TODO migrations
+    # ALTER TABLE build ADD COLUMN last_complete BOOLEAN NOT NULL DEFAULT FALSE;
+    # UPDATE build SET last_complete = TRUE WHERE id IN (SELECT MAX(id) FROM build WHERE state = 3 OR state = 5 GROUP BY package_id);
+    last_complete = Column(Boolean, nullable=False, default=False)
```

Comment:
I would vote for consistency - other booleans are not prefixed with *is*: we have cancel_requested, not is_cancel_requested, deps_resolved not are_deps_resolved and so on.